<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shop_Window</class>
 <widget class="QDialog" name="Shop_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>data/maxresdefault.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="shop_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>52</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Магазин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="check_balance">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 71, 163);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="balance_label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Баланс:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>280</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>370</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 89, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Избражение 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>250</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Minecraft Seven Cyrillic (russi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>350</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 50; 
color: rgb(255, 255, 255);
border : 2px solid white
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>610</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 175, 175);
color: rgb(255, 0, 0);

border-radius : 50; 
border : 3px solid red;</string>
   </property>
   <property name="text">
    <string>Не достаточно средств</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
